### DISCUSSION TITLE
Epiphany icons look really bad

### DISCUSSION THREAD

**Discussion by marcinjahn** [elementary/icons#981] [General]
## Problem

The icons in eOS Epiphany look really bad:

![image](https://user-images.githubusercontent.com/10273406/102475548-4859a180-405a-11eb-9f9f-32b57c819178.png)

Compare it with the "vanilla" Epiphany:

![image](https://user-images.githubusercontent.com/10273406/102475677-75a64f80-405a-11eb-8f1b-08be49563d18.png)

I think that it is clear which one looks better. Especially the "star" and "gear" icons break the "feel". The eOS version lacks consistency and "professional" look.

## Proposal

The icons should be redesigned. Actually even using the "vanilla" version would already be a huge improvement.

## Prior Art

Vanilla Epiphany:
![image](https://user-images.githubusercontent.com/10273406/102475677-75a64f80-405a-11eb-8f1b-08be49563d18.png)

macOS Safari, where the original UI comes from, also does it so much better.

Btw, I submitted a similar problem for the Mail app: https://github.com/elementary/mail/issues/525

**Comment by alice-mkh** [comment]
> macOS Safari, where the original UI comes from, also does it so much better.

Not original, the only shared code we have with Safari is WebKit.

**Reply by marcinjahn** [reply]
I know, I meant the the UI idea comes from Safari :)

**Comment by hanaral** [comment]
Elementary uses colour and visual metaphors for its iconography. If you think that its too childish for you, I'm sure you'll enjoy the wacky sound effects macOS has.

**Reply by marcinjahn** [reply]
Hey, I don't mean to offend anyone! It's just some honest feedback, I love elementary, I use it every day :)
I actually like those "wacky" sound effects, although I liked better the ones prior to Big Sur :D
